kernel/arch/dreamcast/kernel/timer.c

```diff
@@ -1,86 +1,118 @@
 /* KallistiOS ##version##
 
    timer.c
-   Copyright (c)2000, 2001, 2002 Megan Potter
+   Copyright (c) 2000, 2001, 2002 Megan Potter
+   Copyright (c) 2023 Falco Girgis
 */
 
 #include <assert.h>
-
 #include <stdio.h>
+#include <stdint.h>
```

Comment:
You declared the stdint in the header file. You shouldn't have to do it here too.

Reply:
Good point.